pkg/eventprocessor/impl/executor/impl/txnscope_runsql_test.go

```diff
@@ -0,0 +1,555 @@
+package impl
```

Comment:
I've reorganized a bit all these tests:
- Different names
- I think some splits of subtests on separate tests.

Mostly because now we can have "OneEventPerTxn" or multiple (which wasn't true before).

Also, you should expect tests to feel a bit cleaner since:
- We don't do `ValidateCreateTable(..., query)` anymore
- We don't depend on internal results of the previous `Txnprocessor`.